pages/08.scr/default.en.md

```diff
@@ -0,0 +1,42 @@
+---
+title: SysEleven Container Registry
+page-toc:
+    active: false
+body_classes: 'title-center title-h1h2'
+---
+
+[SysEleven Container Registry (SCR)](https://scr.syseleven.de) is the managed registry for containers and Helm charts from [SysEleven](https://www.syseleven.de) - a Berlin based managed hosting company.
+
+[SCR](https://scr.syseleven.de) is based on [Cloud Native Comuputing Foundation](cncf.io) incubated, open source project [Harbor](https://goharbor.io/)
+
+
+## Architectural Overview
+
+The following document explains in detail the architechture of [Harbor](https://goharbor.io/)
+
+![Harbor Architecture Overview](https://github.com/goharbor/harbor/wiki/Architecture-Overview-of-Harbor)
+
+## User Guide
+
+SysEleven Container Registry follows the same user workflow as Harbor, following upstream tutorials can help SCR users
+perform common tasks such as creating and managing new projects, adding repositories to projects, managing image label etc.
+
+### Working with projects
+This section describes how users with the developer, master, and project administrator roles manage and participate in SCR projects.
+- [Creating new Projects](https://github.com/goharbor/harbor/blob/master/docs/1.10/working_with_projects/create_projects.md)
+- [Configuring project properties](https://github.com/goharbor/harbor/blob/master/docs/1.10/working_with_projects/project_configuration.md)
+
+### Working with images and tags
```

Comment:
We should also mention vulnerability scanning See (https://github.com/goharbor/harbor/tree/master/docs/1.10/administration/vulnerability_scanning)